staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/metrics.go

```diff
@@ -310,6 +310,16 @@ var (
 		},
 		[]string{priorityLevel, "success"},
 	)
+	apiserverDispatchWithNoAccommodation = compbasemetrics.NewCounterVec(
```

Comment:
@dgrisonnet - any feedback on this metric?

[I looks reasonable and useful for me, but wanted to get some feedback from SIG-instrumentation before approving.]